deployment/tfhub/object-detection-url.ipynb

```diff
@@ -0,0 +1,643 @@
+        "        return {\n",
+        "            \"method\": \"predict\",\n",
+        "            \"args\": f\"{self.example()}\",\n",
+        "            \"returns\": \"file, has_car, score, ymin, xmin, ymax, xmax\",\n",
+        "            \"description\": \"\"\"\n",
+        "                Identify whether a given object is present in the URL image, with its score and bounding boxes.\n",
+        "            \"\"\",\n",
+        "            \"input_description\": \"\"\"\n",
+        "                A list with image URLs.\n",
+        "            \"\"\",\n",
+        "            \"output_description\": \"\"\"\n",
+        "                A CSV file with information about all the processed images.\n",
+        "                The columns presents the file names, if the object was found (boolean), its score and bounding boxes.\n",
+        "            \"\"\"\n",
+        "        }\n",
+        "    \n",
+        "    def example(self):\n",
+        "        \"\"\"Return example input json-serializable data.\"\"\"\n",
+        "        return [\"000001_0.jpg\", \"http://s3.amazonaws.com/verta-starter/street-view-images/000001_0.jpg\"]"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "5SvthV9gd4lk"
+      },
+      "source": [
+        "## 5. Model Register"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "UR7XtPNOHTgO",
+        "outputId": "17a97dc4-6755-4d3e-f20e-3d0d36d658c8"
+      },
+      "outputs": [],
+      "source": [
+        "registered_model = client.get_or_create_registered_model(name='Object Detection - TF Hub')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "-RctZHJ8MMi5"
+      },
+      "outputs": [],
+      "source": [
+        "input = {\n",
+        "    \"file_name\": \"\",\n",
+        "    \"image_str\": \"\"\n",
+        "}\n",
+        "output = {\n",
+        "    \"file_name\": \"\",\n",
+        "    \"has_car\": False,\n",
+        "    \"score\": 0,\n",
+        "    \"ymin\": 0,\n",
+        "    \"xmin\": 0,\n",
+        "    \"ymax\": 0,\n",
+        "    \"xmax\": 0\n",
+        "}"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "vFlrCL2EWnJR",
+        "outputId": "fc33882d-7ea0-4a94-f9f1-b898446ba27c"
+      },
+      "outputs": [],
+      "source": [
+        "model = registered_model.create_standard_model(\n",
+        "    name = 'url',\n",
+        "    model_cls = DetectObject,\n",
+        "    environment = Python(requirements = ['tensorflow', 'tensorflow_hub', 'dill', 'Pillow']),\n",
+        "    model_api = ModelAPI([input], [output])\n",
```

Comment:
See my comment in the other notebook about `ModelAPI`.

Reply:
Done.